dashboard/test/models/pd/application/facilitator1920_application_test.rb

```diff
@@ -0,0 +1,332 @@
+      @application.expects(:find_default_workshop).never
+
+      @application.assign_default_workshop!
+      assert_equal @fit_workshop.id, @application.reload.pd_workshop_id
+    end
+
+    test 'assign_default_fit_workshop! saves the default fit workshop' do
+      @application.expects(:find_default_fit_workshop).returns(@fit_workshop)
+
+      @application.assign_default_fit_workshop!
+      assert_equal @fit_workshop.id, @application.reload.fit_workshop_id
+    end
+
+    test 'assign_default_fit_workshop! does nothing when a fit workshop is already assigned' do
+      @application_with_fit_workshop.expects(:find_default_fit_workshop).never
+
+      @application_with_fit_workshop.assign_default_fit_workshop!
+      assert_equal @fit_workshop.id, @application_with_fit_workshop.reload.fit_workshop_id
+    end
+
+    test 'fit_workshop returns the workshop associated with the assigned fit workshop id' do
+      assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+    end
+
+    test 'fit_workshop returns nil if the assigned workshop has been deleted' do
+      @fit_workshop.destroy!
+      assert_nil @application_with_fit_workshop.fit_workshop
+    end
+
+    test 'registered_fit_workshop? returns true when the applicant is enrolled in the assigned fit workshop' do
+      create :pd_enrollment, workshop: @fit_workshop, user: @application_with_fit_workshop.user
+      assert @application_with_fit_workshop.registered_fit_workshop?
+    end
+
+    test 'registered_fit_workshop? returns false when the applicant is not enrolled in the assigned fit workshop' do
+      refute @application_with_fit_workshop.registered_fit_workshop?
+    end
+
+    test 'registered_fit_workshop? returns false when no fit workshop is assigned' do
+      @application_with_fit_workshop.update! fit_workshop_id: nil
+      refute @application_with_fit_workshop.registered_fit_workshop?
+    end
+
+    test 'workshop cache' do
+      create :pd_enrollment, workshop: @fit_workshop, user: @application_with_fit_workshop.user
+
+      # Original query: Workshop, Session, Enrollment
+      assert_queries 3 do
+        assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+      end
+
+      # Cached
+      assert_queries 0 do
+        assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+        assert @application_with_fit_workshop.registered_fit_workshop?
+      end
+    end
+
+    test 'workshop cache prefetch' do
+      # Workshop, Session, Enrollment
+      assert_queries 3 do
+        Facilitator1920Application.prefetch_associated_models([@application_with_fit_workshop])
+      end
+
+      assert_queries 0 do
+        assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+
+        # also prefetches assigned workshop
+        assert_equal @workshop, @application_with_fit_workshop.workshop
+      end
+    end
+
+    test 'fit_cohort' do
+      skip "update when 1920 fit registration is implemented"
```

Comment:
Suggest removing this test instead of using the skip

Reply:
I want to keep it so I don't forget and we lose a test - and I am implementing 1920 fit reg tomorrow, so this shouldn't end up checked in and spamming